### PR TITLE
[ONEAPP-27569] SmartSense Motion Sensor: Use same icon as catalog for ST sensors

### PR DESCRIPTION
**Pull request by tpmanley**
The catalog has two entries for SmartThings motion sensors. One entry for the newest sensor and second entry for all the other sensors. The second entry uses a different motion sensor icon so we need to specify UI Metadata that also uses that same icon.

https://smartthings.atlassian.net/browse/ONEAPP-27569

### CODE REVIEW

**Comment by tpmanley** [comment]
This wasn't working before due a plugin issue which has now been addressed in STG. I do need to do a little more testing before this is merged but I think it's ready for review.

**Comment by tpmanley** [comment]
I finished testing so this is ready to merge once it's approved

**Review by dkirker — DISMISSED** [review]

**Comment by dkirker** [comment]
It isn't part of the PR, but line 24 in SmartSense Motion Sensor has some spacing issue with the temp measurement capability declaration. Dunno if you want to grab that, too.

File: devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy:24]: @dkirker fixed

**Review by dkirker — APPROVED** [review]

**Pull request merged**
